zh/lesson-10.md

```diff
@@ -24,8 +24,19 @@ toc-description: "数学模式和数学符号。"
 * 显示模式
 
 ```latex
-\documentclass{article}
-\usepackage[T1]{fontenc}
+% !TEX program=xelatex
+
+% 临时patch，否则使用中文标点，TexLive.net会编译错误
```

Comment:
is there an issue on texlive.net that we could fix there rather than have a patch in the document?  Ssorry I can not read the comment well enough, something about chinese punctuation not working at texlive.net? It shoudl be a standard texlive 2024 installed there, last updated a few days ago

Reply:
Oh sorry, I see, you were needing the latest version of the L3 layer

I just updated texlive on the server ans it works without this patch now

https://texlive.net/run?%25%20!TEX%20program=xelatex%0A%0A%5Cdocumentclass%5BUTF8%5D%7Bctexart%7D%0A%5Cusepackage%7BxeCJK%7D%0A%5Cbegin%7Bdocument%7D%0A%E4%B8%80%E4%B8%AA%E5%8C%85%E5%90%AB%E8%A1%8C%E5%86%85%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F%E7%9A%84%E5%8F%A5%E5%AD%90%EF%BC%9A%24y%20=%20mx%20+%20c%24%E3%80%82%0A%E5%8F%A6%E4%B8%80%E4%B8%AA%E5%8C%85%E5%90%AB%E8%A1%8C%E5%86%85%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F%E7%9A%84%E5%8F%A5%E5%AD%90%EF%BC%9A%245%5E%7B2%7D=3%5E%7B2%7D+4%5E%7B2%7D%24%E3%80%82%0A%0A%E7%AC%AC%E4%BA%8C%E6%AE%B5%E5%8C%85%E5%90%AB%E6%98%BE%E7%A4%BA%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F%E3%80%82%0A%5C%5B%0A%20%20y%20=%20mx%20+%20c%0A%5C%5D%0A%E7%9C%8B%E7%9C%8B%E6%AE%B5%E8%90%BD%E5%A6%82%E4%BD%95%E5%9C%A8%E6%98%BE%E7%A4%BA%E5%85%AC%E5%BC%8F%E5%90%8E%E7%BB%A7%E7%BB%AD%E3%80%82%0A%5Cend%7Bdocument%7D

Reply:
thanks, I will remove all the patches and make another test.